prow/config.yaml

```diff
@@ -553,7 +553,7 @@ presubmits:
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170718-c3574ec6
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
```

Comment:
I don't know the answer here, so I'm asking:
is it expected that we have exactly the same hash for both prow and kubeadm images? Why is that?